tour_config/fields.py

```diff
@@ -9,7 +10,14 @@
 
 class UnixDateTimeField(forms.DateTimeField):
     def to_python(self, value):
-        dt = super(UnixDateTimeField, self).to_python(value)
+        dt = None
```

Comment:
Good catch. I think it would be helpful if we did since it took us a while to figure out what was going on here in the first place.